meetings/2023/2023-11-13.md

```diff
@@ -0,0 +1,38 @@
+# OpenJS Foundation Security Collab Space Meeting 2023-11-13
+
+## Links
+
+* **Recording**:
+* **GitHub Issue**: https://github.com/openjs-foundation/security-collab-space/issues/80
+
+## Present
+
+* bensternthal (@bensternthal)
+* Jordan Harband (@ljharb)
+* Matt Rutkowski
+* Chris de Almeida (@ctcpip)
+* Joe Sepi
```

Comment:
```suggestion
* Matt Rutkowski
* Chris de Almeida (@ctcpip)
* Joe Sepi (@joesepi)
```

Reply:
Thanks! I was just going back through and updating those. :)